templates/zerver/app/index.html

```diff
@@ -59,6 +59,34 @@
     background-color: hsl(212, 28%, 18%);
     color: hsl(236, 33%, 90%);
     }
+
+    .zcontainer{
+     position: relative;
+     margin: auto;
+     display: flex;
+     justify-content: center;
+    }
+    .zlogo{
+     height: 64px;
+     padding:10px;
+    }
+    @keyframes zspinner {
+     to {transform: rotate(360deg);}
+    }
+    .zspinner:before {
+     content: '';
+     box-sizing: border-box;
+     position: absolute;
+     width: 84px;
+     height: 84px;
+     border-radius: 50%;
+     border: 4px solid rgb(240,240,240);
+     border-top-color: rgb(132,132,132);
```

Comment:
Can you switch these colors to HSL? (We lint for that in CSS files, but this is inline CSS because it's intended to run when CSS files haven't loaded properly)